awscodecommit/main.go

```diff
@@ -298,7 +298,7 @@ func (clients Clients) sendPREvent(pullRequest *codecommit.PullRequest) error {
 
 	event := cloudevents.Event{
 		Context: cloudevents.EventContextV03{
-			Type:            "AWS CodeCommit commit event",
+			Type:            "com.amazon.codecommit",
```

Comment:
as a prefix this is probably correct. But is there something we should be adding to the end to make it more specific? E.g. `push` or something like that related to the action? If this is a static string then it's useful to know it's from CodeCommit instead of GitHub but it's not useful to know what type of action caused this event.